src/components/VisualPickerOption/styled/content.js

```diff
@@ -1,9 +1,10 @@
 import styled from 'styled-components';
 import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
 
+const sizeMap = { large: '210px', medium: '142px', small: '100px' };
 const StyledContent = attachThemeAttrs(styled.span)`
-    height: 142px;
-    width: 100%;
+    height: ${props => sizeMap[props.size]};
+    width: ${props => sizeMap[props.size]};
```

Comment:
here we need to do a fallback to a default size since the user can pass a wrong size where `sizeMap[props.size]` will return undefined